### PR TITLE
[zh-cn] Resync admission controllers page

### PR DESCRIPTION
**Pull request by tengqm**
Mainly replacing '/zh' by '/zh-cn'.

### CODE REVIEW

**Review by howieyuen — APPROVED** [review]
/approve

File: content/zh-cn/docs/reference/access-authn-authz/admission-controllers.md [hunk 1]
```diff
@@ -1378,7 +1368,7 @@ You can enable additional admission controllers beyond the default set using the
 ## 有推荐的准入控制器吗？
```
Comment (ydFu) [inline content/zh-cn/docs/reference/access-authn-authz/admission-controllers.md:1368]:
```suggestion
## 有推荐的准入控制器吗？ {#is-there-a-recommended-set-of-admission-controllers-to-use}
```

Comment (ydFu) [reply content/zh-cn/docs/reference/access-authn-authz/admission-controllers.md:1368]: I agree with you.

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**